### ISSUE TITLE
Description text cut off (e.g. 'my sample t...')

### ISSUE THREAD

**Issue by snovik75** [tqdm/tqdm#582]
Why is my description text cut off (e.g. 'my sample t...' instead of 'my sample text') and how to force it to show full length?

**Comment by casperdcl** [comment]
can you post a MWE?

**Comment by snovik75** [comment]
MWE_

**Comment by casperdcl** [comment]
minimal working example

**Comment by AbysmalBiscuit** [comment]
Hi, I am having the same issue when using `tqdm_notebook`.
Here is an example of what I am doing:
```python
from time import sleep
from tqdm import tqdm_notebook
import sys

progress_bar = tqdm_notebook(range(100), desc="This is a long description.", file=sys.stdout)

for i in progress_bar:
    sleep(0.02)
    if i == 50:
        progress_bar.set_description("Half way there!")
        sleep(9999)  # Long sleep to be able to screenshot
```
Screenshots of the resultant output:
![screenshot from 2018-08-28 16-58-16](https://user-images.githubusercontent.com/24716026/44731618-e456ec00-aae3-11e8-9379-ed8567ac3672.png)
![screenshot from 2018-08-28 16-58-17](https://user-images.githubusercontent.com/24716026/44731620-e4ef8280-aae3-11e8-9c7d-4f1cfa7839ef.png)

I have tried looking through the issues to find a solution but nothing worked.

From investigating the cause I have found that it has to with the IntProgress widget, wherein it sets the width of the description to 80px, so when I opened my Developer Tools and disabled it in the CSS the progress bar shrunk in order to accommodate the larger description. I have provided a screenshot (I have added red boxes around the CSS shown in the inspector and progress bar description for emphasis):
![screenshot from 2018-08-29 07-00-14](https://user-images.githubusercontent.com/24716026/44766427-ac8a8b80-ab59-11e8-924c-7cf24034a0b2.png)

In the [ipywidgets styling documentation](https://ipywidgets.readthedocs.io/en/stable/examples/Widget%20Styling.html) they explain [how to handle the description field getting truncated](https://ipywidgets.readthedocs.io/en/stable/examples/Widget%20Styling.html#Description). I have tested both their suggestions (using a Label widget and adjusting the IntProgress widget style so the description fits), both work. It seems like adjusting the widget style leads to behavior which resembles the most how tqdm behaves when used in a terminal.

Here is a screenshot of what it looks like with the style adjustment fix (note: the effect in appearance is exactly the same as when disabling the 80px width):
![screenshot from 2018-08-29 07-42-53](https://user-images.githubusercontent.com/24716026/44767736-5caec300-ab5f-11e8-8b5c-3faa7e69321d.png)

@casperdcl I saw in some issues that you plan to make changes to `tqdm_notebook`, but would you be open to me raising a small PR to fix this truncation issue?

**Comment by casperdcl** [comment]
Yes, happy to accept a PR for this; shouldn't affect any other issues.

**Comment by AbysmalBiscuit** [comment]
Cool, I have opened the PR: #599

**Comment by casperdcl** [comment]
fixed by #599
